Python/bytecodes.c

```diff
@@ -1913,6 +1915,36 @@ dummy_func(
             str = PyStackRef_FromPyObjectSteal(str_o);
         }
 
+        inst(BUILD_INTERPOLATION, (values[4] -- interpolation)) {
+            PyObject *interpolation_o = _PyInterpolation_FromStackRefStealOnSuccess(values);
+            if (interpolation_o == NULL) {
+                ERROR_NO_POP();
+            }
+            INPUTS_DEAD();
+            interpolation = PyStackRef_FromPyObjectSteal(interpolation_o);
+        }
+
+        inst(BUILD_TEMPLATE, (pieces[oparg] -- template)) {
```

Comment:
```suggestion
        inst(BUILD_TEMPLATE, (strings, interpolations -- template)) {
```

A template is a pair of tuple. Let the compiler build the tuples.